README.md

```diff
@@ -113,6 +113,7 @@ require'nvim-tree'.setup { -- BEGIN_DEFAULT_OPTS
       icons = {
         corner = "└ ",
         edge = "│ ",
+        item = "│ ",
```

Comment:
should be empty string by default :)

Reply:
it is used to replace the innerest file's edge icon,  shouldn't be empty

Reply:
i see sorry